src/components/Thanks/SingleVersion/OptInModule.vue

```diff
@@ -114,6 +117,8 @@ const description = computed(() => {
 const loansToDisplay = computed(() => props.loans.slice(0, 3));
 
 const updateOptIn = value => {
+	buttonState.value = 'loading';
```

Comment:
Don't we still need to wait to close the module? I'm assuming right now the loaders are only shown briefly until the modal is closed, and before the saving is complete. We want to show the module with the loaders and only close when saving is completed.

Reply:
We can await for this functions so: https://github.com/kiva/ui/blob/aec98b1a48078f66810ddac2f7d2a29e3eff7445/src/components/Thanks/SingleVersion/OptInModule.vue#L128

Agree?

Reply:
Yeah, we'll want to await both of those functions.